requires-ci.txt

```diff
@@ -7,7 +7,7 @@ flake8==3.9.2
 flaky==3.7.0
 flask-talisman==1.0.0
 isort==4.3.21;python_version<"3.7"
-mimesis
+mimesis<=11.1.0
```

Comment:
NBD since this is only in requires-ci, but why is this lock needed?

Reply:
The latest version introduced a union type like `str | int` and that is only supported in recent python versions.

Reply:
OK yeah weird:
> To work with Mimesis on Python versions 3.8 and 3.9, the final compatible version is Mimesis 11.1.0. Install this specific version to ensure compatibility.

but even v12.1.0 says
> Requires: Python >=3.8, <4.0

Wonder why they didn't just bake that first statement into the `python_requires` 🤔